src/org/usfirst/frc/team694/robot/commands/auton/DrivetrainMoveToLineCommand.java

```diff
@@ -27,13 +27,13 @@ protected void initialize() {
     // Called repeatedly when this Command is scheduled to run
     protected void execute() {
         Robot.drivetrain.updateSensors();
-        Robot.drivetrain.tankDrive((speedMode + 1) * 0.25, (speedMode + 1) * 0.25);
-        isReached = Robot.drivetrain.isOnLine((int) speedMode);
+        Robot.drivetrain.tankDrive((speedNumber + 1) * 0.25, (speedNumber + 1) * 0.25);
+        isReached = Robot.drivetrain.isOnLine((int) speedNumber);
     }
 
     // Make this return true when this Command no longer needs to run execute()
     protected boolean isFinished() {
-        return isReached || (Math.max(Robot.drivetrain.getRightEncoderDistance(),Robot.drivetrain.getLeftEncoderDistance()) > (distance + 5));
+        return isReached || (Math.max(Robot.drivetrain.getRightEncoderDistance(),Robot.drivetrain.getLeftEncoderDistance()) > (distance));
```

Comment:
Use Robot.drivetrain.getEncoderDistance() which grabs the max of both sides for you